tests/integration_tests/functional/test_cmd_line_start.py

```diff
@@ -206,6 +225,73 @@ def test_config_with_default_limit(test_microvm_with_api, vm_config_file):
     assert stdout.encode("utf-8") == response_err.encode("utf-8")
 
 
+@pytest.mark.parametrize(
+    "vm_config_file",
+    ["framework/vm_config_with_mmdsv1.json"]
+)
+def test_config_start_with_mmds_default(test_microvm_with_api, vm_config_file):
+    """
+    Test MMDS V1 behavior when the microvm is configured from file.
+
+    @type: functional
+    """
+    test_microvm = test_microvm_with_api
+
+    _configure_vm_from_json(test_microvm, vm_config_file)
+    _configure_network_interface(test_microvm)
+
+    # Network namespace has already been created.
+    test_microvm.spawn(with_netns=False)
```

Comment:
Why do we need to avoid using a netns ?

Reply:
Since we start the microVM from config file, we need to create the tap device before calling `spawn()` ( so we need to add the network namespace before that). That flag ensures that we don't try to add the netns twice.

Reply:
I see. Makes sense. But can we modify the spawning logic in order to create the netns before starting the VM instead ? I think it would be simpler.

Reply:
`spawn()` already creates the nets before starting the jailer, but this is still problematic when configuring a VM from file. Here is why:
When configuring the VM from API, we call `spawn()`, which creates the netns and starts the jailer. Then, if we want to attach a net device, we first need to create the associated tap, then call the FC API to create the network interface. 
When configuring a VM from a json file, if we want to be able to attach a net device, we need to create the network namespace AND THEN create the tap, ALL BEFORE starting the jailer.

The current approach adds a flag to `spawn()` to disallow the creation of the network namespace, so that we can have better control over it. Instead of letting `spawn()` handle the netns, we create the network namespace and tap, then call `spawn()`.

Another approach would be to embed tap creation into `spawn()`, but I feel like this would be a much more rigid approach. It would also burden the `spawn()` function in case we mean to start a microVM from json file with multiple network devices. WDYT?

Reply:
Ok, got it. It makes sense to keep it as it is.

Reply:
Done.